DxDispatch/src/dxdispatch/CommandLineArgs.cpp

```diff
@@ -83,10 +83,15 @@ CommandLineArgs::CommandLineArgs(int argc, char** argv)
             cxxopts::value<std::string>()->default_value("manual")
         )
         (
-            "f,onnx_free_dim_override",
+            "f,onnx_free_dim_name_override",
             "List of free dimension overrides by name (ONNX models only). Can be repeated. Example: -f foo:3 -f bar:5",
             cxxopts::value<std::vector<std::string>>()->default_value({})
         )
+        (
+            "F,onnx_free_dim_denotation_override",
```

Comment:
>onnx_free_dim_denotation_override [](http://example.com/codeflow?start=15&length=33)

Ah yes, these things. In 4 years, I've never used them because models I diagnose rarely have them, and the names are more reliable, but I suspect they are useful somewhere.